simulator/src/ux.js

```diff
@@ -621,19 +621,35 @@ export function fillSubcircuitElements() {
 async function postUserIssue(message) {
 
     var img = generateImage("jpeg", "full", false, 1, false).split(',')[1];
-    const result = await $.ajax({
-            url: 'https://api.imgur.com/3/image',
-            type: 'POST',
-            data: {
-                image: img
-            },
-            dataType: 'json',
-            headers: {
-                Authorization: 'Client-ID 9a33b3b370f1054'
-            },
-        });
 
-    message += "\n" + result.data.link;
+    let result;
+    try {
+        result = await $.ajax({
+                url: 'https://api.imgur.com/3/image',
+                type: 'POST',
+                data: {
+                    image: img
+                },
+                dataType: 'json',
+                headers: {
+                    Authorization: 'Client-ID 9a33b3b370f1054'
```

Comment:
I don't think we should keep any credentials openly like this

Reply:
It's a Client ID which is supposed to be public

Reply:
Can't we make this like any env variable ?

Reply:
The thing is, this request is sent purely by user (it doesn't hit the backend), so using env variables is not possible